mslib/_tests/test_version.py

```diff
@@ -0,0 +1,35 @@
+# -*- coding: utf-8 -*-
+"""
+
+    mslib._tests.utils
+    ~~~~~~~~~~~~~~~~~~
+
+    This module provides common functions for MSS testing

```

Comment:
what is the purpose of the test? This should be in this line. 

When you have imported a module, the doc string is accessible by the `__doc__` method

```
import test_version
print(test_version.__doc__)
```

Our IDEs shows than this information too.